quipucords/api/scan/model.py

```diff
@@ -84,7 +84,7 @@ class ScanOptions(models.Model):
     JBOSS_BRMS_EXT = 'jboss_brms_ext'
 
     max_concurrency = models.PositiveIntegerField(default=50)
-    disable_optional_products = \
+    disabled_optional_products = \
         models.ForeignKey(DisableOptionalProductsOptions,
```

Comment:
Not gonna update this class name, DisableOptionalProductsOptions?

Reply:
Done.  All reference updated.